### PR TITLE
Bump arbitrary to 1.2

### PR DESCRIPTION
**Pull request by lnicola**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/main/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

File: geo-types/Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ use-rstar_0_9 = ["rstar_0_9", "approx"]
 
 [dependencies]
 approx = { version = ">= 0.4.0, < 0.6.0", optional = true }
-arbitrary = { version = "=1.1.3", optional = true }
+arbitrary = { version = "1.2", optional = true }
```
Comment (michaelkirk) [inline geo-types/Cargo.toml:24]: I was convinced to [prefer precise version constraints](https://users.rust-lang.org/t/psa-please-specify-precise-dependency-versions-in-cargo-toml/71277).

Comment (lnicola) [inline geo-types/Cargo.toml:24]:
```suggestion
arbitrary = { version = "1.2.0", optional = true }
```

**Comment by lnicola** [comment]
Don't r+ this, I'll squash tomorrow.

**Comment by michaelkirk** [comment]
bors d=lnicola

**Comment by lnicola** [comment]
bors r=michaelkirk

**Pull request merged**